_posts/en/newsletters/2024-04-10-newsletter.md

```diff
@@ -0,0 +1,331 @@
+branch][64bit arith cscriptnum]."
+a3link="https://bitcoincore.reviews/29221#l-67" %}
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [HWI 3.0.0][] is a release of the next version of this package
+  providing a common interface to multiple different hardware signing
+  devices.  The only significant change in this release is that emulated
+  hardware wallets will no longer be automatically detected; see the
+  description of [HWI #729][] below for details.
+
+- [Core Lightning 24.02.2][] is a maintenance release that fixes "a
+  [small incompatibility][core lightning #7174]" between Core Lightning's
+  and LDK's implementation of a particular part of the LN gossip
+  protocol.
+
+- [Bitcoin Core 27.0rc1][] is a release candidate for the next major
+  version of the network's predominant full node implementation.
+  Testers are encouraged to review the list of [suggested testing
+  topics][bcc testing].
+
+
+## Notable code and documentation changes
+
+_Notable recent changes in [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], [Lightning BOLTs][bolts repo],
+[Bitcoin Inquisition][bitcoin inquisition repo], and [BINANAs][binana
+repo]._
+
+*Note: the commits to Bitcoin Core mentioned below apply to its master
+development branch and so those changes will likely not be released
+until about six months after the release of the upcoming version 27.*
+
+- [Bitcoin Core #29648][] removes libconsensus after it was previously
+  deprecated (see [Newsletter #288][news288 libconsensus]).
+  Libconsensus was an attempt to make Bitcoin Core's consensus logic
+  usable in other software.  However, the library hasn't seen any
+  significant adoption and it has become a burden on maintenance of
+  Bitcoin Core.
+
+- [Bitcoin Core #29130][] adds two new RPCs.  The first will generate a
+  [descriptor][topic descriptors] for a user based on the settings they want and then add
+  that descriptor to their wallet.  For example, the following command
+  will add support for [taproot][topic taproot] to an old wallet created
+  without that support:
+
+  ```
+  bitcoin-cli --rpcwallet=mywallet createwalletdescriptor bech32m
+  ```
+
+  The second RPC is `gethdkeys` (get [HD][topic bip32] keys), which will
+  return each xpub used by the wallet and (optionally) also each xpriv.
+  When a wallet contains multiple xpubs, the particular one to use can
+  be indicated when calling `createwalletdescriptor`.
+
+- [LND #8159][] and [#8160][lnd #8160] add experimental (disabled by
```

Comment:
hey opetch! sorry to be this week's "um, actually" but https://github.com/lightningnetwork/lnd/pull/8160 adds forwarding support to LND so that it can be used by other nodes in the network for blinded path creation. It's just disabled because we need https://github.com/lightningnetwork/lnd/pull/8485 to be fully spec-compliant with error handling (NB for blinded path privacy), and it'll be turned on by default once that goes through.

Sending support (on some APIs) was added in https://github.com/lightningnetwork/lnd/pull/7267, though admittedly the PR title is terrible so it's not particularly clear.